### PR TITLE
Attempt to improve print keys to work with running and stopped miner

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

**Comment by Vagabond** [comment]
```
(base) ^andrew@system76:: ./_build/dev/rel/miner/bin/miner ping
pong
(base) ^andrew@system76:: ./_build/dev/rel/miner/bin/miner print_keys
{pubkey,"1129aa2VNz5HNMeZgRZwhqZg2AWFyqmmzY8aqrr8FdgXEbV1oE9K"}.
{onboarding_key,undefined}.
{animal_name,"rhythmic-laurel-seagull"}.
(base) ^andrew@system76:: ./_build/dev/rel/miner/bin/miner stop
ok
(base) ^andrew@system76:: ./_build/dev/rel/miner/bin/miner ping
Node 'miner@127.0.0.1' not responding to pings.
(base) _andrew@system76:: ./_build/dev/rel/miner/bin/miner print_keys
{pubkey,"1129aa2VNz5HNMeZgRZwhqZg2AWFyqmmzY8aqrr8FdgXEbV1oE9K"}.
{onboarding_key,undefined}.
{animal_name,"rhythmic-laurel-seagull"}.
```

File: rebar.config [hunk 1]
```diff
@@ -172,7 +172,7 @@
                                        {copy, "scripts/extensions/info", "bin/extensions/info"},
                                        {copy, "scripts/extensions/dkg", "bin/extensions/dkg"},
                                        {copy, "scripts/extensions/authorize", "bin/extensions/authorize"},
-                                       {template, "scripts/extensions/print_keys", "bin/extensions/print_keys"},
+                                       {copy, "scripts/extensions/print_keys", "bin/extensions/print_keys"},
```
Comment (fvasquez) [inline rebar.config]: That's right.  No more templating.

File: scripts/extensions/print_keys [hunk 2]
```diff
@@ -4,8 +4,8 @@ CONFIG=$(find $ROOTDIR/releases -name sys.config)
 LIB_DIR=$ROOTDIR/lib
 
 if relx_nodetool "ping" > /dev/null 2>&1; then
-    cat /var/data/public_keys
-    exit 0
+    relx_nodetool rpc miner_keys print_keys #| sed '$d'
```
Comment (fvasquez) [inline scripts/extensions/print_keys]: Is the `#` in front of `|` a typo?

File: src/miner_keys.erl [hunk 3]
```diff
@@ -70,11 +81,9 @@ keys({ecc, Props}) when is_list(Props) ->
...
 %% friendly way to stdout. This is used by other services (like
 %% gateway_config) to get read access to the public keys
-print_keys(ConfigFile) ->
-    {ok, [Config]} = file:consult(ConfigFile),
-    {blockchain, BlockchainConfig} = proplists:lookup(blockchain, Config),
-    BaseDir = proplists:get_value(base_dir, BlockchainConfig, "data"),
-    KeyConfig = case proplists:get_value(key, BlockchainConfig, undefined) of
+print_keys(_) ->
+    BaseDir = application:get_env(blockchain, base_dir, "data"),
+    KeyConfig = case application:get_env(blockchain, key, undefined) of
```
Comment (fvasquez) [inline src/miner_keys.erl]: I see what you did here. Need to verify this works in the non-rpc case.

**Comment by fvasquez** [comment]
@Vagabond this looks like the way forward to me so I'm going to go ahead and merge this PR into mine.

**Pull request merged**